ci/do_ci.sh

```diff
@@ -296,22 +297,25 @@ elif [[ "$CI_TARGET" == "bazel.compile_time_options" ]]; then
     TEST_TARGETS=("@envoy//test/...")
   fi
   # Building all the dependencies from scratch to link them against libc++.
-  echo "Building and testing with wasm=wasmtime: ${TEST_TARGETS[*]}"
-  bazel_with_collection test "${BAZEL_BUILD_OPTIONS[@]}" --define wasm=wasmtime "${COMPILE_TIME_OPTIONS[@]}" -c dbg "${TEST_TARGETS[@]}" --test_tag_filters=-nofips --build_tests_only
+  echo "Building and testing with wasmtime: ${TEST_TARGETS[*]}"
+  bazel_with_collection test "${BAZEL_BUILD_OPTIONS[@]}" "${COMPILE_TIME_OPTIONS[@]}" \
+    --//source/extensions/wasm_runtime/wasmtime:enabled \
+    --//source/extensions/wasm_runtime/wavm:enabled=false \
```

Comment:
Things seem a bit more magical now; whereas before we explicitly enabled, now it's by default?

Reply:
L288